.github/workflows/create-github-release.yaml

```diff
@@ -0,0 +1,82 @@
+name: "Create Github Release"
+
+# NOTE: This workflow does not work well if not triggered by a tag push due to
```

Comment:
Remove double negative: "NOTE: This workflow only works well when triggered by a tag push..."